robotintelligence.cpp

```diff
@@ -100,3 +139,11 @@ double RobotIntelligence::random(double lower_bound, double upper_bound) {
 	std::default_random_engine re;
 	return unif(re);
 }
+
+double RobotIntelligence::gaussian(double x, double mean, double sigma) {
+    if(sigma == 0)
+        return x==mean ? 1 : 0;

```

Comment:
this weight should be normalized to the gaussian function for sigma in limit zero, so actually it should be infinity : 0